File: frontend/src/app/models/Flight.ts
export interface Flight {
  id: number;
  dateOfFlight: Date;
  startingPlace: string;
  landingPlace: string;
  ticketPrice: number;
  ticketPriceVip: number;
  availabilityOfPlaces: number;
}

export interface FlightWithoutId {
  dateOfFlight: string;
  startingPlace: string;
  landingPlace: string;
  ticketPrice: string;
  ticketPriceVip: string;
  availabilityOfPlaces: string;
}

File: astrosales/src/main/java/com/github/rodis00/astrosales/flight/entity/Flight.java
package com.github.rodis00.astrosales.flight.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

import java.time.LocalDateTime;

@Entity
@Data
@Table(name = "flight")
public class Flight {
    @Id
    @GeneratedValue
    private Integer id;

    private LocalDateTime dateOfFlight;

    private String startingPlace;

    private String landingPlace;

    private float ticketPrice;

    private float ticketPriceVip;

    private int availabilityOfPlaces;
}
